docs/_docs/customizingyourgateway.md

```diff
@@ -104,8 +104,13 @@ Or you might want to mutate the response messages to be returned.
 1. Write a filter function.
    ```go
    func myFilter(ctx context.Context, w http.ResponseWriter, resp proto.Message) error {
-   	w.Header().Set("X-My-Tracking-Token", resp.Token)
-   	resp.Token = ""
+        t, ok := resp.(*external.Tokenizer)
```

Comment:
Maybe even something like `externalpb` to make it more clear what general type this is.
```suggestion
        t, ok := resp.(*externalpb.Tokenizer)
```

Reply:
@johanbrandhorst - yeah I like this. I wondered if we should place the type right above the filter?